packages-node/providence-analytics/src/program/utils/optimised-glob.js

```diff
@@ -219,7 +238,12 @@ export async function optimisedGlob(globOrGlobs, providedOptions = {}) {
     options.onlyDirectories = true;
   }
 
-  const globs = Array.isArray(globOrGlobs) ? Array.from(new Set(globOrGlobs)) : [globOrGlobs];
+  const regularGlobs = Array.isArray(globOrGlobs) ? toUniqueArray(globOrGlobs) : [globOrGlobs];
```

Comment:
Did you mean to use ensureArray here?

Reply:
Also, toUniqueArray here seems not necessary as we do it anyway on L246.

Reply:
Thanks. Fixed 👍